src/libData/AccountData/Transaction.cpp

```diff
@@ -207,11 +207,11 @@ bool Transaction::operator>(const Transaction& tran) const {
   return tran < *this;
 }
 
-Transaction& Transaction::operator=(const Transaction& src) {
-  copy(src.m_tranID.asArray().begin(), src.m_tranID.asArray().end(),
-       m_tranID.asArray().begin());
-  m_signature = src.m_signature;
-  m_coreInfo = src.m_coreInfo;
+// Transaction& Transaction::operator=(const Transaction& src) {
```

Comment:
Is this function unused? Maybe we can just remove completely.

Reply:
I think this will be removed when I rebase to current master.